hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -1261,8 +1260,19 @@
           ]
         }
       }
-    ],
+    ]
     // Debug bus attachment
-    dbg : "regs",
+    dbg: [
+      { window: {
+          name: "dbg"
+          // 0x0 ... 0x400 word addresses are allocated here.
+          items: "1024"
+          swaccess: "rw",
+          data-intg-passthru: "true",
+          byte-write: "false",
+          desc: '''Access window to lc_ctrl CSRs and .'''
+        }
+      },
+    ]
```

Comment:
should these LC_CTRL changes go in the previous commit?

Reply:
They could probably all three be squashed -- I initially started with code you had written and wanted to try to preserve authorship. But we can just squash them all and attribute the whole thing to both of us.

Reply:
Yeah that sounds sensible to me.